src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
```

Comment:
I now see that you moved it to separate files. But you forgot to delete this code from file where it was before. Please delete unnecessary code.